src/components/inputs/ChipInput.jsx

```diff
@@ -358,6 +371,29 @@ const ChipInput = React.forwardRef(function ChipInputFn(
 	);
 
 	const onFocus = useCallback(() => setIsActive(true), []);
+
+	const onPaste = useCallback(
+		(e) => {
+			if (createChipOnPaste) {
+				const pastedText = e.clipboardData.getData('Text');
+				const separatorsRegex = new RegExp(pasteSeparators.join('|'), 'gi');
+				const reducedChips = reduce(
+					pastedText.split(separatorsRegex),
+					(acc, v) => {
+						if (trim(v) !== '') acc.push(trim(v));
+						return acc;
+					},
+					[]
+				);
+				savePastedValue(uniq(reducedChips));
+				setTimeout(() => {
+					contentEditableInput.current.innerHTML = '';
+				}, 10);
```

Comment:
Not sure about this timeout: if it is meant to avoid to have text content inside the input, what about a e.preventDefault() to prevent the default behaviour of the paste event?

Reply:
> Not sure about this timeout: if it is meant to avoid to have text content inside the input, what about a e.preventDefault() to prevent the default behaviour of the paste event?

Thanks I didn't knew about this solution 😇